schema/json/schema-16.0.19.json

```diff
@@ -2750,4 +2750,4 @@
       "type": "array"
     }
   }
-}
+}
```

Comment:
This was modified during the merge/rebase of the old branch which is why the warning fired.